docker-compose.yaml

```diff
@@ -0,0 +1,46 @@
+version: "3.3"
+services:
+  star-tides:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    environment:
+      FLASK_DEBUG: 1
+      PYTHONUNBUFFERED: 1
+    #env_file:
```

Comment:
Remove commented out code.

Reply:
Removed the `environment` tag and uncommented `env_file`.